src/modules/sidebar/sidebar.component.scss

```diff
@@ -17,24 +26,33 @@
     &:hover {
       background-color: $stache-color-gray-light;
     }
-
-    &.active,
-    &.active:hover {
-      background-color: $stache-color-blue;
-      color: #fff;
-    }
   }
 
-  .stache-nav-sidebar .stache-nav-sidebar {
-    background-color: $stache-color-gray-light;
-    padding: 0; 
-    margin-bottom: 8px;
+  .stache-nav-sidebar {
+    .stache-nav-sub {
+      background-color: $stache-color-gray-light;
+      padding: 0; 
+      margin-bottom: 8px;
+      display: none;
```

Comment:
Well, the css is handling the display of the submenus. As it stands, all submenus are available, but only the active submenus are visible to the end user. This should give folks flexibility to show those other navs if needed.